internal/p2p/peermanager.go

```diff
@@ -589,6 +596,11 @@ func (m *PeerManager) Dialed(address NodeAddress) error {
 		return fmt.Errorf("peer %q was removed while dialing", address.NodeID)
 	}
 	now := time.Now().UTC()
+	if peer.Inactive {
+		m.options.Metrics.PeersInactivated.Add(-1)
+	}
+	peer.Inactive = false
```

Comment:
Why would we dial a peer that is `Inactive`? Wouldn't that imply some inconsistency?

Reply:
But I don't think they make `Ranked` right? So we'd never dial them